webapp/public/locales/en/main.json

```diff
@@ -418,7 +419,19 @@
   "study.modelization.clusters.minUpTime": "Min up time",
   "study.modelization.clusters.minDownTime": "Min down time",
   "study.modelization.clusters.spinning": "Spinning",
-  "study.modelization.clusters.co2": "CO2",
+  "study.modelization.clusters.co2": "CO\u2082",
```

Comment:
study.modelization.clusters.thermal ?

study.modelization.clusters is used for thermal and renewables